README.md

```diff
@@ -57,6 +57,21 @@ Each Issue has a Maintainer that is the "supervisor" for the general topic the i
 7. In order to debug your jvm code, you should start godot with command line `--jvm-debug-port=XXXX`, where `XXXX`
 stands for the jmx port of you choice. You can then setup remote debug configuration in Intellij' Idea.
 
+## Command line arguments
+
+In order to debug, or override behaviour of this module, there are some command lines arguments you can add when
+starting godot:  
+| Command line argument               | Description                                                                                                                | Usage                                                                                        |
```

Comment:
Personally i find the usage section obsolete. IMO it just duplicates whats i  the first two sections

Reply:
I find that it does not permit any ambiguity about the usage.  
We have commands with `=` and others without.  
Usage section is here so that user know the different ways to use the command.

Reply:
ok then